### ISSUE TITLE
Choice of "Fraud" vs "Abuse" wording.

### ISSUE THREAD

**Issue by dvorak42** [antifraudcg/antifraudcg.github.io#2]

**Comment by pbjorke** [comment]
Can we also change the name of the working group to "anti-abuse" (instead of antifraud)?

Fraud can get tangled up in more of a legal definition/interpretation, abuse is a more general/broader term that is more suitable for this context.

**Comment by michaelficarra** [comment]
@pbjorke I agree that "abuse" would be more general and apply to more topics that we would consider in-scope but not typically refer to as "fraud". However, I think it would imply to the general passerby that many other topics are in-scope which we do not want to include. So while both terms are imperfect, I think "fraud" is closer, and any further clarification can be garnered by reading the charter.

**Comment by AramZS** [comment]
I think the way to think about it is that we are looking for use cases from people who consider those use cases to be fraud or abuse. Lots of words have legal definitions and if we avoided every word that had a legal definition somewhere we'd have a very short dictionary to choose from. I don't have a problem with the name because I do think it references a *specific technical space* that exists on the web by convention and through specific and present marketing around that space. More than that I think there is value in that we want this to be a space where people with use cases that are concerned with fraud to come to and therefore this is a clear indicator to me that this is the right name.

**Comment by sysrqb** [comment]
Currently, the charter defines the mission as `identify and define scenarios involving fraud and unwanted traffic` - this seems to highlight the current tension between fraud and abuse. I generally agree with @AramZS, however if this CG is focused on "fraud" then other forms of "abuse" should be considered out of scope. From the application/user-agent side, I am more concerned with working on anti-abuse mechanisms (of which I include anti-fraud, but only as a subclass).

If this group is planning on discussing "unwanted traffic" that falls outside of the definition of fraud, then I recommend using "abuse" (or another more general term), and then clearly defining the scope in the charter. If this group will primarily focus on fraud (and anti-fraud mechanisms), then I agree with the other comments.

**Comment by dvorak42** [comment]
To try unblocking the charter, and then nailing down specific scope in parallel (and amending the charter later), I've made PR #6 (https://github.com/antifraudcg/antifraudcg.github.io/pull/6) to incorporate some of the abuse wording, PTAL.
